synapse/tools/autodoc.py

```diff
@@ -217,9 +244,21 @@ def processFormsProps(rst, dochelp, forms, univ_names):
         link = f'.. _dm-form-{name.replace(":", "-")}:'
         rst.addHead(hname, lvl=2, link=link)
 
+        baseline = f'The base type for the form can be found at :ref:`dm-type-{name.replace(":", "-")}`.'
         rst.addLines(doc,
+                     '',
+                     baseline,
                      '')
 
+        ex = dochelp.formhelp.get(name)
+        if ex:
+            rst.addLines('',
+                         f'A example of ``{name}``{raw_back_slash_colon}',
```

Comment:
'An example'

Reply:
Some of these aren't working right, for example:
https://synapse.docs.vertex.link/en/feature_autodoc_stormtypes/synapse/autodocs/datamodel_types.html#file-mime
or
https://synapse.docs.vertex.link/en/feature_autodoc_stormtypes/synapse/autodocs/datamodel_types.html#crypto-currency-address